### PR TITLE
trayer: add page

### LINKED ISSUE #10440: page request: trayer
### Command description

I would like an addition of a tldr page for [trayer](https://github.com/sargon/trayer-srg).

Trayer is a systray for UNIX desktop.

### Command details

trayer

### OS

- [ ] Android
- [X] Linux
- [ ] macOS
- [ ] sunOS
- [ ] Windows

### PR DESCRIPTION
**Pull request by abhiyandhakal**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.1.8-3

---

Closes #10440.

### CODE REVIEW

File: pages/linux/trayer.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# trayer
+
+> A lightweight GTK-2 based systray for UNIX desktop.
```
Comment (kbdharun) [inline pages/linux/trayer.md:3]:
```suggestion
> A lightweight GTK-2 based systray.
```
IMO we don't need to specify `for UNIX desktop` as we are moving away from this naming in other platform pages.

File: pages/linux/trayer.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://github.com/sargon/trayer-srg>.
+
+- Run trayer:
```
Comment (kbdharun) [inline pages/linux/trayer.md:6]:
```suggestion
- Run `trayer`:
```

File: pages/linux/trayer.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`trayer`
+
+- Position trayer to the top:
+
+`trayer --edge {{top}}`
```
Comment (kbdharun) [inline pages/linux/trayer.md:12]:
```suggestion
- Position `trayer` to a specific edge:

`trayer --edge {{left|right|top|bottom}}`
```

File: pages/linux/trayer.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Provide height as 32px and width as 10% of screen resolution:
+
+`trayer --width {{10}} height {{ 32 }}`
```
Comment (kbdharun) [inline pages/linux/trayer.md:16]:
```suggestion
`trayer --width {{10}} --height {{32}}`
```
Shouldn't the height be a flag too here?

File: pages/linux/trayer.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Run trayer:
+
+`trayer`
+
+- Position trayer to the top:
+
+`trayer --edge {{top}}`
+
+- Provide height as 32px and width as 10% of screen resolution:
```
Comment (kbdharun) [inline pages/linux/trayer.md:14]:
```suggestion
- Provide a specific height and width of the panel (in pixels):
```
I would suggest having the command description a bit more generalised

File: pages/linux/trayer.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`trayer --width {{10}} height {{ 32 }}`
+
+- Provide width in pixel:
+
+`trayer --widthtype {{pixel}} --width {{72}}`
+
+- Align trayer to the right:
+
+`trayer --align {{right}}`
```
Comment (kbdharun) [inline pages/linux/trayer.md:24]:
```suggestion
- Align `trayer` to a specific direction:

`trayer --align {{left|center|right}}`
```

File: pages/linux/trayer.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Provide icon gap 10px:
```
Comment (kbdharun) [inline pages/linux/trayer.md:26]:
```suggestion
- Provide space between icons (in pixels):
```

File: pages/linux/trayer.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Position trayer to the top:
+
+`trayer --edge {{top}}`
+
+- Provide height as 32px and width as 10% of screen resolution:
+
+`trayer --width {{10}} height {{ 32 }}`
+
+- Provide width in pixel:
```
Comment (kbdharun) [inline pages/linux/trayer.md:18]:
```suggestion
- Provide the width of the panel in pixels or percentages:
```

File: pages/linux/trayer.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+`trayer --widthtype {{pixel}} --width {{72}}`
```
Comment (kbdharun) [inline pages/linux/trayer.md:20]:
```suggestion
`trayer --widthtype {{pixel|percent}} --width {{72}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. I have some suggestions for this page.

**Tip:** _You can group all these suggestions in the Files changed tab and make a single commit._

File: pages/linux/trayer.md (hunk 4 above)
Comment (abhiyandhakal) [reply pages/linux/trayer.md:16]: yes... I forgot. Thanks for that

**Review by abhiyandhakal — COMMENTED** [review]
I think this looks good

**Review by kbdharun — APPROVED** [review]

File: pages/linux/trayer.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+- Provide the width of the panel in pixels or percentages:
+
+`trayer --widthtype {{pixel|percent}} --width {{72}}`
+
+- Align `trayer` to a specific direction:
+
+`trayer --align {{left|center|right}}`
+
+- Provide space between icons (in pixels):
```
Comment (acuteenvy) [inline pages/linux/trayer.md:26]:
```suggestion
- Provide spacing between icons (in pixels):
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**